src/Flunt.Br/Document/Voter.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Text.RegularExpressions;
+using Flunt.Br.Document.interfaces;
+using System.Collections.Generic;
+
+namespace Flunt.Br.Document
+{
+    internal class Voter : IValidate
```

Comment:
@marcoslcosta essa validação é para titulo de eleitor? Se for, acredito que poderia se chamar VoterDocument ou VoterTitle.